composeml/data_slice.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+
+
+class Context:
```

Comment:
What if we just called this `DataSliceMetadata` or `DataSliceContext`?

Reply:
That works. I like `DataSliceContext`.